components/gallery/GalleryItem.vue

```diff
@@ -161,6 +165,11 @@ useNuxt2Meta({
   font-size: 2.4375em;
 }
 
+hr {
+  height: 1px;
+  margin: 60px 0;
```

Comment:
classname`my-x` may help

Reply:
I dont think my-x is a class?

Reply:
> I dont think my-x is a class?

cf Bulma [doc](https://bulma.io/documentation/helpers/spacing-helpers/) example: `my-5`
please don't overload `hr` elements

Reply:
@roiLeo my-5 is still not enough spacing to match what we have in figma

Reply:
What are you trying to fix here?
Instead of adding margin to `hr` try to add spacing on element around the component